contracts/src/v0.8/VRFV2Wrapper.sol

```diff
@@ -343,6 +344,13 @@ contract VRFV2Wrapper is ConfirmedOwner, TypeAndVersionInterface, VRFConsumerBas
     return weiPerUnitLink;
   }
 
+  /**
+   * @dev Calculates premium needed for running an assembly call() post-EIP150.
```

Comment:
Doc nit: `Calculates extra amount of gas required for running ...`

Reply:
```suggestion
   * @dev Calculates extra amount of gas required for running an assembly call() post-EIP150.
```